app/models/assessment.rb

```diff
@@ -9,17 +9,23 @@ class Assessment < ApplicationRecord
   validates :version, inclusion: { in: CFEConstants::VALID_ASSESSMENT_VERSIONS, message: "not valid in Accept header" }
 
   has_one :applicant, dependent: :destroy
-  has_one :capital_summary, dependent: :destroy
-  has_one :gross_income_summary, dependent: :destroy
-  has_one :disposable_income_summary, dependent: :destroy
+  has_one :capital_summary, dependent: :destroy, class_name: "ApplicantCapitalSummary"
+  has_one :gross_income_summary, dependent: :destroy, class_name: "ApplicantGrossIncomeSummary"
+  has_one :disposable_income_summary, dependent: :destroy, class_name: "ApplicantDisposableIncomeSummary"
+
+  has_one :partner, dependent: :destroy
+  has_one :partner_capital_summary, dependent: :destroy
+  has_one :partner_gross_income_summary, dependent: :destroy
+  has_one :partner_disposable_income_summary, dependent: :destroy
```

Comment:
Correct, Rails-y 'convention over configuration' means unless you less it otherwise, the 🐍 -case relationships are assumed to map to 🐫 -case class names.